docs/core/porting/third-party-deps.md

```diff
@@ -1,38 +1,31 @@
 ---
-title: Porting to .NET Core - Analyzing your Third-Party Party Dependencies
-description: Porting to .NET Core - Analyzing your Third-Party Dependencies
-keywords: .NET, .NET Core
+title: Porting to .NET Core - Analyzing your third-party dependencies
+description: Learn how to analyze third-party dependencies in order to port your project from .NET Framework to .NET Core.
 author: cartermp
 ms.author: mairaw
-ms.date: 06/20/2016
+ms.date: 01/17/2018
 ms.topic: article
 ms.prod: .net-core
 ms.devlang: dotnet
 ms.assetid: b446e9e0-72f6-48f6-92c6-70ad0ce3f86a
 ---
+# Porting your code to .NET Core - Analyzing your third-party dependencies
 
-# Porting to .NET Core - Analyzing your Third-Party Party Dependencies
+If you're looking to port your code to .NET Core, the first step in the porting process is to understand your third-party dependencies. Evaluate which of them, if any, don't yet run on .NET Core and develop a contingency plan for these dependencies.
 
-The first step in the porting process is to understand your third party dependencies.  You need to figure out which of them, if any, don't yet run on .NET Core, and develop a contingency plan for those which don't run on .NET Core.
+## Analyzing NuGet packages
 
-## Prerequisites
+Analyzing NuGet packages for portability is easy. A NuGet package is itself a set of folders that contain platform-specific assemblies. So, what you have to do is check to see if there's a folder that contains a compatible assembly inside the package.
 
-This article will assume you are using Windows and Visual Studio, and that you have code which runs on the .NET Framework today.
+The easiest way to inspect NuGet Package folders is to use the [NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer) tool. After downloading it, use the following steps to see the folder names:
```

Comment:
Nit: the primary way of distributing NPE is the Windows Store, which means you don't really download it, you install it. So I would say:

> After installing it, use the following steps to see the folder names: